### PR TITLE
New docker-compose.yml for developing purposes

### PR DESCRIPTION
**Pull request by jazio**
### Description

Added a docker-compose.yml for quickly building the platform on AWS or local pc without the need of LAMP.

### Change log

- Added: docker-compose.yml
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands
From root of the project.
docker-compose up -d

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+services:
+  web:
+    image: fpfis/httpd-php-dev:5.6
+    working_dir: ${PWD}
+    volumes:
+      - ${PWD}:${PWD}
+    environment:
+      - DOCUMENT_ROOT=${PWD}/build
+      - XDEBUG_CONFIG=idekey=cloud9ide remote_connect_back=0 remote_host=${DOCKER_HOST_IP} remote_enable=1 remote_port=9000
+      - ~/.composer/cache:/root/.composer/cache
```
Comment (drupol) [inline docker-compose.yml]: This needs to be removed.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    depends_on:
+      - mysql
+    ports:
+      - 8080:8080
+
+  mysql:
+    image: percona/percona-server:5.6
```
Comment (drupol) [inline docker-compose.yml]: 5.6 or 5.7 ?

**Review by drupol — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (jazio) [reply docker-compose.yml]: Rather 5.6 [here](https://github.com/ec-europa/cloud9/search?q=percona&unscoped_q=percona).
I've push changes.

**Comment by Fefaine** [comment]
See https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-2490 
The file is provided in C9, submit your change requests to https://github.com/ec-europa/cloud9/blob/master/salt/docker/files/docker-compose/docker-compose-all.yml

**Pull request closed without merging**